src/dotty/language.scala

```diff
@@ -4,6 +4,16 @@ object language {
 
   class Feature
 
+  /** Enable scala.Dynamic */
```

Comment:
The change on #1550 will break this PR. I will have to find another way to handle the members in the language module.